core-bundle/src/Resources/contao/controllers/BackendPassword.php

```diff
@@ -127,6 +133,46 @@ public function run()
 
 		return $objTemplate->getResponse();
 	}
+
+	/**
+	 * Evaluate custom regex. See https://github.com/contao/contao/issues/5316
+	 *
+	 * @param  string      $pw The password
+	 * @return bool|string The error message, false otherwise
+	 */
+	private function _evalRegexp($pw)
+	{
+		// no custom regex available
+		if (!isset($GLOBALS['TL_HOOKS']['addCustomRegexp']) || !\is_array($GLOBALS['TL_HOOKS']['addCustomRegexp']))
+		{
+			return false;
+		}
```

Comment:
I think this needs to be removed (or changed to `empty($GLOBALS['TL_DCA']['tl_user']['fields']['password']['eval']['rgxp'])`) as `$GLOBALS['TL_DCA']['tl_user']['fields']['password']['eval']['rgxp']` can, in theory, also be one of the built-in checks.

Reply:
lgtm, will change it in the next version (note that I'm refactoring here an existing pull request from @richardhj)